image_match/signature_database_base.py

```diff
@@ -159,6 +159,8 @@ def __init__(self, k=16, N=63, n_grid=9,
                 considering how much variance to keep in the image (default (5, 95))
             distance_cutoff (Optional [float]): maximum image signature distance to
                 be considered a match (default 0.45)
```

Comment:
So the default value of `9.0` is the reason the test is failing ... only 8 words match, it looks like.  Maybe I need to think a bit more about the default value, but if you change to something less than 8, the tests should pass.